Scripts/DCS-BIOS/lib/modules/aircraft_modules/L-39.lua

```diff
@@ -703,4 +703,26 @@ L_39:defineFloat("BACK_CANOPY_POS", 140, { 0, 1 }, "Gauges", "BACK Door Position
 
 L_39:defineReadWriteRadio("R832M_RADIO", 19, 7, 3, 1000, "R-832M Radio")
 
+L_39:defineIndicatorLight("STORE_LEFT", 618, "Warning, Caution and IndicatorLights", "Front Left Stores Lamp (green)")
```

Comment:
Control ID in this module starts with ```FRONT_``` or ```BACK_``` where applicable. Also the description should start with either ```Front``` or ```Back```.